datalad/support/tests/test_annexrepo.py

```diff
@@ -694,11 +706,12 @@ def test_AnnexRepo_get_file_backend(src, dst):
     eq_(ar.get_file_backend('test-annex.dat'), 'SHA1')
 
 
-@known_failure_windows
 @with_tempfile
 def test_AnnexRepo_always_commit(path):
 
     repo = AnnexRepo(path)
+    if repo.is_managed_branch():
+        raise SkipTest('Test logic invalid for adjusted repos')
```

Comment:
@kyleam in #5325 introduced `@skip_if_adjusted_branch` which should be used here instead of the conditional

Reply:
Done.